util/purl/image.go

```diff
@@ -4,8 +4,8 @@ import (
 	"strings"
 
 	"github.com/containerd/containerd/platforms"
-	distreference "github.com/docker/distribution/reference"
-	"github.com/opencontainers/go-digest"
+	"github.com/docker/distribution/reference"
```

Comment:
btw, the reason I used the specific name was that there is also

https://github.com/containerd/containerd/tree/main/reference and
https://github.com/moby/moby/tree/master/reference

Reply:
Yeah that makes sense, I think we should make that change everywhere then at some point, instead of just here.

Reply:
Happy to do that in a follow-up, if this is a blocker :+1: